radixdlt-java/src/main/java/com/radixdlt/client/core/Bootstrap.java

```diff
@@ -2,19 +2,28 @@
 
 import com.radixdlt.client.core.address.RadixUniverseConfig;
 import com.radixdlt.client.core.address.RadixUniverseConfigs;
+import com.radixdlt.client.core.network.RadixNetworkEpic;
 import com.radixdlt.client.core.network.bootstrap.NodeFinder;
 import com.radixdlt.client.core.network.RadixNode;
+import com.radixdlt.client.core.network.epics.DiscoverNodesEpic;
+import com.radixdlt.client.core.network.epics.DiscoverSingleNodeEpic;
 import io.reactivex.Observable;
+import java.util.Collections;
+import java.util.List;
 import java.util.function.Supplier;
 
 public enum Bootstrap implements BootstrapConfig {
-	BETANET(
+	LOCALHOST(
 		RadixUniverseConfigs::getBetanet,
 		Observable.just(
 			new RadixNode("localhost", false, 8080),
 			new RadixNode("localhost", false, 8081)
 		)
 	),
+	LOCALHOST_SINGLENODE(
```

Comment:
Why are these different?

Reply:
SINGLE_NODE will always connect to the same node so tests which have two clients can be synced easily